core/src/main/java/com/graphhopper/util/PathMerger.java

```diff
@@ -183,4 +182,53 @@ private void calcAscendDescend(final PathWrapper rsp, final PointList pointList)
         rsp.setAscend(ascendMeters);
         rsp.setDescend(descendMeters);
     }
-}
\ No newline at end of file
+
+    private String unconnectedListString(List<Path> paths) {
+        StringBuilder sb = new StringBuilder().append("[");
+
+        for (int n = 0; n < paths.size(); ++n) {
+            if (!paths.get(n).isFound())
+                sb.append(String.format("(%d,%d), ", n, n+1));
+        }
+
+        if (sb.length() < 2)
+            throw new IllegalArgumentException("list doesn't have unavailable paths");
+
+        sb.setLength(sb.length() - 2); // delete last occurrence of  ", "
+        sb.append("]");
+
+        return sb.toString();
+    }
+
+    private Map<String, Object> unconnectedInfo(PathWrapper altRsp, List<Path> paths) {
+        PointList waypoints = altRsp.getWaypoints();
+
+        ArrayList<Object> idxFrom = new ArrayList<>();
```

Comment:
Do we need 6 ArrayLists here? Wouldn't it be easier to have everything as map so that the json result looks nice as well.

```
details = {
  "connection-not-found-1": {
    "idFrom": 1,
    ...
  },
  "connection-not-found-2": {
    "idFrom": 2
    .....
  }
}
```